tests/containers/docker_runc.pm

```diff
@@ -24,43 +25,82 @@ sub run {
     select_serial_terminal;
 
     my ($running_version, $sp, $host_distri) = get_os_release;
-    my $docker = $self->containers_factory('docker');
-    my @runtimes = ();
-    push @runtimes, "runc" if (is_leap(">15.1") or !is_sle('=15'));
+    my $runc = "runc";    # runc executable
 
-    record_info 'Setup', 'Setup the environment';
-    # runC cannot create or extract the root filesystem on its own. Use Docker to create it.
-
-    # create the rootfs directory
+    # Runtime setup and installation
+    record_info 'Test #1', 'Installation and test preparation';
+    if (script_run("which $runc") != 0) {
+        if (is_transactional) {
+            select_console 'root-console';
+            trup_call("pkg install $runc");
+            check_reboot_changes;
+        } else {
+            zypper_call("in $runc");
+        }
+    }
+    record_info("$runc", script_output("$runc -v"));
+    # Create root filesystem for the test container. We need docker for this preparation step.
     assert_script_run('mkdir rootfs');
```

Comment:
We're no longer removing this directory in `cleanup()`;

Reply:
Good catch.

Reply:
I'm also adding a `rm -rf rootfs` here, so the test would not fail in case the directory already exists.